auth/jwt_validator.py

```
@@ -0,0 +1,108 @@
+                    "verify_aud": False,  # можно временно отключить, если BFF шлёт токены не для auth
+                    "verify_iss": True,
+                    "verify_exp": True,
+                },
+            )
+            return payload
+
+        except jwt.ExpiredSignatureError:
+            raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="Token has expired",
+            )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)